Tao_Current.txt

```diff
@@ -163,133 +187,159 @@ The IETF discussion list is unmoderated. This means that all can express their o
 
 The heart of an IETF meeting is the WG meetings themselves. Different WGs chairs have very different styles, so it is impossible to generalize how a WG meeting will feel. Even though nearly all WGs have agendas for their meetings, some meetings stick tightly to their agenda while others are run more loosely.
 
-There are a few important things that are true for all WG meetings at an IETF meeting. Near the beginning of the meeting, the chair will pass around the "blue sheets", which are paper forms on which everyone prints their name and puts their email address. These are used for long-term archival purpose to show how many people came to a particular meeting and, in rare cases, exactly who showed up. The normal etiquette is to watch where the blue sheets came from and to pass them along in the same direction.
+There are a few important things that are true for all WG meetings at an IETF meeting. Near the beginning of the meeting, the chair will pass around the "blue sheets", which are paper forms on which everyone writes their name and their email address. These are used for long-term archival purpose to show how many people came to a particular meeting and, in rare cases, exactly who showed up. The normal etiquette is to watch where the blue sheets came from and to pass them along in the same direction.
 
 When speaking in a meeting, you should always go to the microphones in the room. For controversial topics, there will be a line at the mic, but do not hesitate to be the first person at the mic if you have a question or a contribution to the discussion. The WG chair or presenter will indicate when you can speak. Although it would be easier to just raise your hand from where you are sitting, the mics perform a very useful task: they let the people listening remotely and in the room hear your question or comment. It is also expected that you will say your name at the mic so that the person taking minutes will know who is speaking.
+
 3.6 Seeing Spots Before Your Eyes
 
 Some of the people at the IETF will have a little colored dot on their name tag. A few people have more than one. These dots identify people who are silly enough to volunteer to do a lot of extra work. The colors have the meanings shown here.
-Color 	Meaning
-Blue 	Working Group/BOF Chair
-Green 	Host group
-Red 	IAB member
-Yello 	IESG member
-Orange 	Nominating Committee member
-Purple 	IAOC
-Pink 	IRSG
-Teal 	RSE
+
+| Color  | Meaning                     |
+|--------|-----------------------------|
+| Blue   | Working Group/BOF Chair     |
+| Green  | Host group                  |
+| Red    | IAB member                  |
+| Yellow | IESG member                 |
+| Orange | Nominating Committee member |
+| Purple | IAOC                        |
+| Pink   | IRSG                        |
 
 (Members of the press wear orange-tinted badges.)
 
 Local hosts are the people who can answer questions about the terminal room, restaurants, and points of interest in the area.
```

Comment:
Local hosts answer questions about the terminal room? It's probably been a long time since that was true.

Reply:
Removed